File: CsFormsFlashcards/CsFormsFlashcards/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;
using System.IO;
namespace CsFormsFlashcards
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();

            //papas con queso chedard
        }

        private void label1_Click(object sender, EventArgs e)
        {

        }

        private void button2_Click(object sender, EventArgs e)
        {
            Agregar frm = new Agregar();
            frm.Show();

        }

        private void button1_Click(object sender, EventArgs e) //boton de empezar
        {
            panel1.Visible = true;
            label2.Visible = true;

            texto testo = new texto();
            Imagen iami = new Imagen();
            testo.obtenerdatos();
            iami.obtenerdatos();

            //MessageBox.Show(iami.CargarImagen());

            label3.Text = testo.mostrarpregunta();
            label4.Text = testo.mostrarrespeusta();

            try
            {
                pictureBox1.Image = Image.FromFile(iami.CargarImagen());
            }
            catch (Exception z)
            { 
            }


        }

        private void Form1_Load(object sender, EventArgs e)
        {
            panel1.Visible = false;
            label2.Visible = false;

            label4.Visible = false;
        }

        private void label2_Click(object sender, EventArgs e)
        {
            panel1.Visible = false;
            label2.Visible = false;

        }

        private void button4_Click(object sender, EventArgs e)
        {
            label4.Visible = true;
            pictureBox1.Visible = false;
        }

        private void button3_Click(object sender, EventArgs e)
        {
            pictureBox1.Visible = true;
            label4.Visible = false;


            texto testo = new texto();
            Imagen iami = new Imagen();
            testo.obtenerdatos();
            iami.obtenerdatos();

            label3.Text = testo.mostrarpregunta();
            label4.Text = testo.mostrarrespeusta();

            try
            {
                pictureBox1.Image = Image.FromFile(iami.CargarImagen());
            }
            catch (Exception zz)
            {
            }

        }
    }
}

File: VbFormsFlashcards/VbFormsFlashcards/Form1.vb
Public Class Form1
    Private Sub Button2_Click(sender As Object, e As EventArgs) Handles Button2.Click

        Dim frm As New Agregar()
        frm.Show()
    End Sub

    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click

        Panel1.Visible = True
        Label1.Visible = True

        Dim testo As New texto()
        Dim iami As New Imagen()
        testo.obtenerdatos()
        iami.obtenerdatos()

        Label3.Text = testo.mostrarpregunta()
        Label4.Text = testo.mostrarrespeusta()

        Try

            PictureBox1.Image = Image.FromFile(iami.CargarImagen())

        Catch ex As Exception

        End Try






    End Sub

    Private Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        Panel1.Visible = False
        Label1.Visible = False

        Label4.Visible = False

    End Sub

    Private Sub Label1_Click(sender As Object, e As EventArgs) Handles Label1.Click
        Panel1.Visible = False
        Label1.Visible = False




    End Sub

    Private Sub Button3_Click(sender As Object, e As EventArgs) Handles Button3.Click

        Label4.Visible = True
        PictureBox1.Visible = False

    End Sub

    Private Sub Button4_Click(sender As Object, e As EventArgs) Handles Button4.Click

        Panel1.Visible = True
        Label1.Visible = True
        Label4.Visible = False

        Dim testo As New texto()
        Dim iami As New Imagen()
        testo.obtenerdatos()
        iami.obtenerdatos()

        Label3.Text = testo.mostrarpregunta()
        Label4.Text = testo.mostrarrespeusta()



        Try

            PictureBox1.Image = Image.FromFile(iami.CargarImagen())
        Catch ex As Exception

        End Try


    End Sub
End Class
